api/models/comments.model.js

```diff
@@ -25,3 +25,12 @@ exports.insertComment = (review_id, username, body) => {
       return response.rows[0];
     });
 };
+
+exports.removeAComment = (commentId) => {
+  const deleteCommentStr = `
+  DELETE FROM comments
+  WHERE comment_id = $1`;
+  return db.query(deleteCommentStr, [commentId]).then((response) => {
+    return response.rows;
```

Comment:
Just needs an extra check/query to send back to 404 if the id doesnt exist that they've tried to delete.